src/man/sssd.conf.5.xml

```diff
@@ -2914,7 +2914,9 @@ ldap_user_extra_attrs = phone:telephoneNumber
         <para>
             Some options used in the domain section can also be used in the
             trusted domain section, that is, in a section called
-            <quote>[domain/<replaceable>DOMAIN_NAME</replaceable>/<replaceable>TRUSTED_DOMAIN_NAME</replaceable>]</quote>.
+	    <quote>[domain/<replaceable>DOMAIN_NAME</replaceable>/<replaceable>TRUSTED_DOMAIN_NAME</replaceable>]</quote>.
+            Where DOMAIN_NAME is actual joined-to base domain. Please refer
```

Comment:
"Where DOMAIN_NAME is actual joined-to (...)" -> "Where DOMAIN_NAME is the actual joined-to (...)"

I'm not an English native speaker, so those things is always confusing for me. In the sentence "Please refer examples below" ... isn't this supposed to be "refer to examples below"?